### PR TITLE
fix(server): fix bootstrap adaptation

### PR DESCRIPTION
**Pull request by xxxrenegade**
- i to em
- add generateAlert (alert-info)
- add bootstrap panel
- add table-responsive
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/Pascalmh%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/176%23issuecomment-70720624%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-01-20T19%3A52%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2086f4cc75a92f6334f2a5ddf9c1a134823aae7ee1%20server.php%2029%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/176%23discussion_r23131984%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Please%20use%20generateErrorBox%20instead%20of%20generateAlert%22%2C%20%22created_at%22%3A%20%222015-01-18T09%3A23%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22Aber%20sollte%20an%20dieser%20stelle%20nicht%20eher%20eine%20alert-info%20alert%20Box%20ausgegeben%20werden%20anstatt%20eine%20alert-warning%20Box%21%3F%22%2C%20%22created_at%22%3A%20%222015-01-18T09%3A55%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222015-01-18T09%3A59%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20server.php%3AL52-70%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/176%23issuecomment-70720624%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/176%23discussion_r23131984%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/176%23discussion_r23132143%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/176%23discussion_r23132166%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/Pascalmh'></a>

- [x] <a href='#crh-comment-Pull 86f4cc75a92f6334f2a5ddf9c1a134823aae7ee1 server.php 29'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/176#discussion_r23131984'>File: server.php:L52-70</a></b>
- <a href='https://github.com/Phhere'></a> Please use generateErrorBox instead of generateAlert
- <a href='https://github.com/xxxrenegade'></a> Aber sollte an dieser stelle nicht eher eine alert-info alert Box ausgegeben werden anstatt eine alert-warning Box!?

<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/176?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/176?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/176?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/176'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by derchrisuk** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/176'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by xxxrenegade** [comment]
:shipit:

**Comment by Phhere** [comment]
@xxxrenegade The small hamster icon should only be used by team members because it approves the pull request :P

**Comment by xxxrenegade** [comment]
OK sollte aber denke ich mal soweit passen ^^

**Comment by xxxrenegade** [comment]
:+1:

File: server.php [hunk 1]
```diff
@@ -53,19 +52,18 @@
...
         }
         $servername = htmloutput($ds[ 'name' ]);
         $info = htmloutput($ds[ 'info' ]);
         eval("\$server = \"" . gettemplate("server") . "\";");
         echo $server;
-        $i++;
     }
 } else {
-    echo $_language->module[ 'no_server' ];
+	echo generateAlert($_language->module[ 'no_server' ], 'alert-info');
```
Comment (Phhere) [inline server.php]: Please use generateErrorBox instead of generateAlert

Comment (xxxrenegade) [inline server.php]: Aber sollte an dieser stelle nicht eher eine alert-info alert Box ausgegeben werden anstatt eine alert-warning Box!?

Comment (Phhere) [inline server.php]: :+1:

**Comment by Pascalmh** [comment]
:shipit::gun:

**Comment by Pascalmh** [comment]
:shipit:

**Pull request closed without merging**